src/Compiler/Driver/CompilerImports.fs

```diff
@@ -2152,6 +2152,20 @@ and [<Sealed>] TcImports
         node {
             CheckDisposed()
 
+            // TODO inject top-down from FSharpChecker
+            let runInParallel =
+                Environment.GetEnvironmentVariable("FCS_PARALLEL_PROJECTS_ANALYSIS")
+                |> bool.TryParse
+                |> function
+                    | true, runInParallel -> runInParallel
+                    | false, _ -> false
+
+            let runMethod =
+                if runInParallel then
+                    NodeCode.Parallel
+                else
+                    NodeCode.Sequential
```

Comment:
We could do that. Do you know if the same (sequential mode in debug) done for other potentially parallel steps like parallel file checking within a project?

Reply:
> We probably want always Sequential if DEBUG is defined.

I'd probablyvote for a separate flag or other way to control that. I can imagine a situation where we'd want to debug FCS with parallel analysis enabled.

Reply:
> We could do that. Do you know if the same (sequential mode in debug) done for other potentially parallel steps like parallel file checking within a project?

Not sure about this one tbh. 

> I'd probably vote for a separate flag or other way to control that. I can imagine a situation where we'd want to debug FCS with parallel analysis enabled.

Sure, I'm fine with other easy way of switching it on and off. I'm mostly concerned that the more things we parallelize, the harder it will be to debug, especially on big projects, so we need a easy to use mechanism to turn those things off.

Reply:
ATM the flag can be set by either an environment flag or at `FSharpChecker` construction/FSC CLI.
Will that be enough for debugging scenarios?